frontend/public/style/_overrides.scss

```diff
@@ -512,3 +512,357 @@ kbd {
     display: none;
   }
 }
+
+// temporary td width's for demo only, responsive pf4 css to follow
+@media (min-width: $screen-md-min){
+  .pf-c-table .meta-status {
+    min-width: 100px;
+  }
+}
+
+// pf4 table spacings adjusted to match existing styles
+.pf-c-table tr > * {
+  --pf-c-table-cell--PaddingLeft: 16px;
+  --pf-c-table-cell--PaddingRight: 16px;
+  --pf-c-table--m-compact-cell--first-last-child--PaddingLeft: 16px;
+  --pf-c-table--m-compact-cell--first-last-child--PaddingRight: 16px;
+}
+
+.pf-c-table.pf-m-compact .pf-c-table__action {
+  --pf-c-table__action--PaddingLeft: 16px;
+  --pf-c-table__action--PaddingRight: 16px;
+  --pf-c-table__action--PaddingTop: 8px;
+  --pf-c-table__action--PaddingBottom: 8px;
+
+  .pf-c-dropdown__toggle {
+    font-size: $font-size-base;
+  }
+  .pf-c-dropdown__menu-item {
+    font-size: $font-size-base;
+  }
+}
+
+// pf table overrides
+.pf-c-table {
+  table-layout: fixed;
+}
+
+.pf-col-kebab { 
+  @media screen and (min-width: 992px) {
+    width: 5% !important;
+  }
+  width: 10% !important;
+  padding-left: 0px !important;
+}
+
+// Responsive 12 column width table classes - pf-u-w-*
+// 1/12 cols
+.pf-u-w-8-on-sm {
+  @media screen and (min-width: 576px) and (max-width: 768px) {
+      width: 8.333% !important;
+  }
+}
+
+.pf-u-w-8-on-md {
+  @media screen and (min-width: 768px) and (max-width: 992px) {
+      width: 8.33% !important;
+  }
+}
+
+.pf-u-w-8-on-lg {
+  @media screen and (min-width: 992px) and (max-width: 1200px) {
+      width: 8.33% !important;
+  }
+}
+
+.pf-u-w-8-on-xl {
+  @media screen and (min-width: 1200px) {
+      width: 8.33% !important;
+  }
+}
+
+// 2/12 cols
+
+.pf-u-w-16-on-sm {
```

Comment:
If these styles are under review, can I give feedback?

These classes make you do a lot more math than the Bootstrap equivalents like `col-sm-2` because you have to divide larger numbers and the numbers don't divide cleanly.

* `col-sm-2`: 12 / 2 = 6 (easy)
* `pf-u-w-16-on-sm`: 100 / 16 = 6.25 (?)

Also the names are longer and would be a lot easier to get wrong or typo.

Reply:
yes - we can certainly rename these to something more like Bootstrap. Agreed 👍

Reply:
converted to `pf-m-[1-12]-col-on-[sm-md-lg-xl]` classes respectively